tests/libslic3r/test_clipper_utils.cpp

```diff
@@ -93,82 +93,82 @@ SCENARIO("Various Clipper operations - xs/t/11_clipper.t", "[ClipperUtils]") {
             }
         }
     }
-	GIVEN("Clipper bug #96 / Slic3r issue #2028") {
-		Slic3r::Polyline subject{
-			{ 44735000, 31936670 }, { 55270000, 31936670 }, { 55270000, 25270000 }, { 74730000, 25270000 }, { 74730000, 44730000 }, { 68063296, 44730000 }, { 68063296, 55270000 }, { 74730000, 55270000 },
-			{ 74730000, 74730000 }, { 55270000, 74730000 }, { 55270000, 68063296 }, { 44730000, 68063296 }, { 44730000, 74730000 }, { 25270000, 74730000 }, { 25270000, 55270000 }, { 31936670, 55270000 },
-			{ 31936670, 44730000 }, { 25270000, 44730000 }, { 25270000, 25270000 }, { 44730000, 25270000 }, { 44730000, 31936670 } };
-		Slic3r::Polygon clip { {75200000, 45200000}, {54800000, 45200000}, {54800000, 24800000}, {75200000, 24800000} };
-        Slic3r::Polylines result = Slic3r::intersection_pl(subject, { clip });
-		THEN("intersection_pl - result is not empty") {
-			REQUIRE(result.size() == 1);
-		}
-	}
-	GIVEN("Clipper bug #122") {
-		Slic3r::Polyline subject { { 1975, 1975 }, { 25, 1975 }, { 25, 25 }, { 1975, 25 }, { 1975, 1975 } };
-		Slic3r::Polygons clip { { { 2025, 2025 }, { -25, 2025 } , { -25, -25 }, { 2025, -25 } },
-								{ { 525, 525 }, { 525, 1475 }, { 1475, 1475 }, { 1475, 525 } } };
-		Slic3r::Polylines result = Slic3r::intersection_pl({ subject }, clip);
-		THEN("intersection_pl - result is not empty") {
-			REQUIRE(result.size() == 1);
-			REQUIRE(result.front().points.size() == 5);
-		}
-	}
-	GIVEN("Clipper bug #126") {
-		Slic3r::Polyline subject { { 200000, 19799999 }, { 200000, 200000 }, { 24304692, 200000 }, { 15102879, 17506106 }, { 13883200, 19799999 }, { 200000, 19799999 } };
-		Slic3r::Polygon clip { { 15257205, 18493894 }, { 14350057, 20200000 }, { -200000, 20200000 }, { -200000, -200000 }, { 25196917, -200000 } };
-		Slic3r::Polylines result = Slic3r::intersection_pl(subject, { clip });
-		THEN("intersection_pl - result is not empty") {
-			REQUIRE(result.size() == 1);
-		}
-		THEN("intersection_pl - result has same length as subject polyline") {
-			REQUIRE(result.front().length() == Approx(subject.length()));
-		}
-	}
+        GIVEN("Clipper bug #96 / Slic3r issue #2028") {
```

Comment:
Most .cpp files use 4-space indentation, so if you are going to adjust indentation, please match that.

Reply:
I've fix the indenting of this using spaces too..

Reply:
Also, thanks for mentioning .clang-format, I forgot to look for that and it does at least reflect the majority of the code being no-tabs-indentation 4-spaces-per-level. Will make it easier to get the code formatter approved.

Reply:
In some of the other projects I've worked on we added custom cmake targets so you could `make tidy` to ensure all the files are systematically formatted. An example of a bunch of cmake targets for doxygen generation, code format, clang-tidy checking, iwyu checking and fixing etc are here;

https://github.com/dbaarda/librsync/blob/ee3df5c8775d571871170c52613c36af1a51db36/CMakeLists.txt#L215C1-L276C1

We then invoke those make targets in the github actions here;

https://github.com/dbaarda/librsync/blob/master/.github/workflows/lint.yml

We don't have it automatically applying and submitting fixes, just checking them. No particular reason for that other than we were a bit paranoid about automatic submits and wanted a bit more manual control.